### DISCUSSION TITLE
Xrip no enlaza con Free parcheada

### DISCUSSION THREAD

**Discussion by martosa1976** [NightscoutFoundation/xDrip#3693] [Bug reports]
La marca del movil es un Redmi Note 11 Pro. La versión del xdrip es "dfcbe80-2024.09.17"
La fuente de datos es "Other App", a pesar de que la fuente de datos seleccionada es "Libre (Patched App)".
En principio he revisado todos los pasos a seguir y están en regla.
Gracias por vuestra ayuda.

**Comment by Navid200** [comment]
Is the sensor Libre?

**Reply by martosa1976** [reply]
Si. Es Sensor es libre 2

**Reply by PhilTU** [reply]
OK. "other app" is normal. To the right of the system status screen (classic) you should see a Libre 2 Patched app status screen. Does it show serial number of sensor?

Can you explain what is the problem, apart from the name used.

**Reply by martosa1976** [reply]
Si. Pone el número de serie 3MH0156ADW0
Parece como si xdrip no reconoce cuando se ha iniciado. Para xdrip gastan 4 días de sensor y en el libre link (parcheada y oficial) le quedan 11 días. No sabemos cuál es el error. Hemos revisado todas las indicaciones, reiniciado pero no se engancha. Gracias

**Reply by martosa1976** [reply]
Acabamos de comprobar que efectivamente el sensor que pone el xdrip es el anterior. No le ha cogido el nuevo. Se ha quedado pillado en el anterior.

**Reply by PhilTU** [reply]
Was the new sensor startwd with patched app?

I noticed you had used official app as well.

**Reply by martosa1976** [reply]
Yes. We started the sensor first with the pached and then Witherspoon the oficial one. We need to use the oficial one because the doctor wnat to se the data from the oficial and if we don't use the oficial one in Spain the don't guve us the sensors free. We are thinking about to change the sensor becasue we are not sleeping. Any advise on the versión of xdrip?. I am algo thinking about build nighscout  but not sure if I have enough digital skills.many thanks for your support

**Reply by PhilTU** [reply]
Ok. Version of patched app I have used sends data to Libreview.

The patched app is old, 2019, and on modern phones sometimes it stops working.

If you have EU sensor there is direct connection method. Using OOP2 to decode. You could try this now with current sensor. https://xdrip.readthedocs.io/en/latest/install/libre2/ 

There appears to be a failure of data exchange from patched app to xDrip+ if the two have different sensor serial numbers.

**Reply by martosa1976** [reply]
Thank you very much. I will keep this instructions

**Comment by Navid200** [comment]
@PhilTU Can you have a look at this please?  There is a translate button that works for me.  I hope it works for you too if you need it.
